README.rst

```diff
@@ -277,7 +277,7 @@ these issues by setting additional Clang C compiler flags as follows:
 Dependencies
 ^^^^^^^^^^^^
 
-Falcon depends on `six` and `python-mimeparse`. `python-mimeparse` is a
+Falcon depends on `python-mimeparse`. `python-mimeparse` is a
```

Comment:
If my opinion is considered here, I'm pretty against this approach: I can't see any benefits of doing this, only drawbacks. Vendoring is only acceptable in special cases where you can't rely on standard mechanisms, like when `pip` vendors some packages because it can't use `pip` to install it (chicken-egg problem).

I can see benefits of dropping `six` because there isn't much of code that need to be compatible, and because Python 2 will reach its end of life soon.

I also want to state that discussion about dropping `python-mimeparse` from dependencies is out of scope of this PR.

Reply:
@tribals absolutely! This is definitely outside of scope of this PR, this was just for further discussions. I will move the discussion to our development chat: https://gitter.im/falconry/dev

Reply:
Cool!

Reply:
@tribals I totally agree with your thoughts on vendoring packages.

But yet again, OTOH, it could be compelling for `python-mimeparse` since the whole thing seems to be just ~190 lines of Python: https://github.com/dbtsai/python-mimeparse/blob/master/mimeparse.py